src/components/analyses/ShareWithSupportDialog.js

```diff
@@ -0,0 +1,577 @@
+/**
+ *
+ * @author Sriram, psarando
+ *
+ */
+import React from "react";
+
+import { useTranslation } from "i18n";
+import PropTypes from "prop-types";
+import { Trans } from "react-i18next";
+
+import ids from "./ids";
+
+import constants from "../../constants";
+
+import { intercomShow } from "common/intercom";
+
+import analysisStatus from "components/models/analysisStatus";
+import ExternalLink from "components/utils/ExternalLink";
+import UtilIds from "components/utils/ids";
+
+import { build, formatDate } from "@cyverse-de/ui-lib";
+
+import {
+    Button,
+    Checkbox,
+    CircularProgress,
+    Dialog,
+    DialogActions,
+    DialogContent,
+    DialogTitle,
+    FormControl,
+    FormLabel,
+    FormControlLabel,
+    Grid,
+    Link,
+    Radio,
+    RadioGroup,
+    TextField,
+    Typography,
+} from "@material-ui/core";
+
+function AnalysisInfo(props) {
+    const { analysis, name, email } = props;
+
+    const { t } = useTranslation("analyses");
+
+    if (analysis) {
+        return (
+            <Grid container spacing={3} style={{ marginTop: 2, fontSize: 12 }}>
+                <Grid item xs={6}>
+                    {t("analysis")}: {analysis.name}
+                </Grid>
+                <Grid item xs={12}>
+                    {t("analysisId")} : {analysis.id}
+                </Grid>
+                <Grid item xs={12}>
+                    {t("app")} : {analysis.app_name}
+                </Grid>
+                <Grid item xs={12}>
+                    {t("currentStatus")} : {analysis.status}
+                </Grid>
+                <Grid item xs={12}>
+                    {t("outputFolder")} : {analysis.resultfolderid}
+                </Grid>
+                <Grid item xs={12}>
+                    {t("startDate")} : {formatDate(analysis.startdate)}
+                </Grid>
+                <Grid item xs={12}>
+                    {t("endDate")} : {formatDate(analysis.enddate)}
+                </Grid>
+                <Grid item xs={12}>
+                    {t("user")} : {analysis.username}
+                </Grid>
+                <Grid item xs={12}>
+                    {t("name")} : {name}
+                </Grid>
+                <Grid item xs={12}>
+                    {t("email")} : {email}
+                </Grid>
```

Comment:
We can also use `utils/GridLabelValue.js` (for all the `label: value` type display) here for consistency.

Reply:
Looks much better with `GridLabelValue`:
![GridLabelValue](https://user-images.githubusercontent.com/996408/94629976-26c9f480-0279-11eb-9f0e-763f66bc30d3.png)